service/src/services/domain/builders/collaboration-card-created/collaboration.card.created.notification.builder.ts

```diff
@@ -0,0 +1,124 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { INotificationBuilder } from '@core/contracts';
+import { CollaborationCardCreatedEventPayload } from '@alkemio/notifications-lib';
+import { UserPreferenceType } from '@alkemio/client-lib';
+import {
+  AlkemioUrlGenerator,
+  NotificationBuilder,
+  RoleConfig,
+} from '@src/services/application';
+import { NotificationTemplateType } from '@src/types';
+import { User } from '@core/models';
+import { ALKEMIO_URL_GENERATOR } from '@common/enums';
+import { EmailTemplate } from '@common/enums/email.template';
+import { CollaborationCardCreatedEmailPayload } from '@common/email-template-payload';
+import { NotificationEventType } from '@alkemio/notifications-lib';
+
+@Injectable()
+export class CollaborationCardCreatedNotificationBuilder
+  implements INotificationBuilder
+{
+  constructor(
+    private readonly notificationBuilder: NotificationBuilder<
+      CollaborationCardCreatedEventPayload,
+      CollaborationCardCreatedEmailPayload
+    >,
+    @Inject(ALKEMIO_URL_GENERATOR)
+    private readonly alkemioUrlGenerator: AlkemioUrlGenerator
+  ) {}
+  build(
+    payload: CollaborationCardCreatedEventPayload
+  ): Promise<NotificationTemplateType[]> {
+    const roleConfig: RoleConfig[] = [
+      {
+        role: 'admin',
+        preferenceType: UserPreferenceType.NotificationAspectCreatedAdmin,
+        emailTemplate: EmailTemplate.COLLABORATION_CARD_CREATED_ADMIN,
+      },
+      {
+        role: 'user',
+        preferenceType: UserPreferenceType.NotificationAspectCreated,
+        emailTemplate: EmailTemplate.COLLABORATION_CARD_CREATED_MEMBER,
+      },
+    ];
+
+    const templateVariables = {
+      hubID: payload.journey.hubID,
+      challengeID: payload.journey?.challenge?.id ?? '',
+      opportunityID: payload.journey?.challenge?.opportunity?.id ?? '',
+      entityID:
```

Comment:
Same principle as suggested before, `journeyID` instead of `entityID`
```suggestion
      journeyID:
```